app/views/hyrax/users/_vitals.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="list-group-item">
+  <span class="glyphicon glyphicon-time" aria-hidden="true"></span> Joined on <%= user.created_at.to_date.strftime("%b %d, %Y") %>
+</div>
+
+<div class="list-group-item">
+  <span class="badge"><%= number_of_collections(user) %></span>
+  <span class="glyphicon glyphicon-folder-open" aria-hidden="true"></span>
+  <%= link_to 'Collections Created', main_app.search_catalog_path(f: {generic_type_sim: ['Collection'], depositor_ssim: [user.to_s]}) %>
```

Comment:
added a comment. it was part of a small commit that only fixed the two links. In Hyrax 2.5 they were broken. This was already fixed in master, in a way that didn't seem to back port, so we fixed it with an override so that we didn't diverge.  I've added a comment to this effect.